services/galley/src/Galley/API/Federation.hs

```diff
@@ -499,15 +502,15 @@ onUserDeleted origDomain udcn = do
               let botsAndMembers = convBotsAndMembers conv
               removeUser (qualifyAs lc conv) (tUntagged deletedUser)
               void $
-                notifyConversationAction
-                  False
-                  (sing @'ConversationLeaveTag)
-                  untaggedDeletedUser
-                  False
-                  Nothing
-                  (qualifyAs lc conv)
-                  botsAndMembers
-                  ()
+                mapError mapFederationError $
```

Comment:
Let's see if this works better with polysemy's run error, we can log the impossible error just to keep our conscience clear and drop it from the returning either. (I hope this comment doesn't sound like gibberish in a week)